icu4j/main/core/src/main/java/com/ibm/icu/message2/MFParser.java

```diff
@@ -589,6 +605,7 @@ private MFDataModel.Message getComplexMessage() throws MFParseException {
             // complex-message   = *(declaration [s]) complex-body
             checkCondition(cp != EOF, "Expected a quoted pattern or .match; got end-of-input");
             MFDataModel.Pattern pattern = getQuotedPattern();
+            skipOptionalWhitespaces(); // Trailing whitespace is allowed
```

Comment:
Same, ebnf in a comment.
---
In general what I did with the ebnf in comments was to grep them, and compare the result with the official ebnf, to see what needs updating.
And to "prove" that I am following the spec to the letter (as much as I could :-)

Reply:
Done